### PR TITLE
Fix: Referring to hyperparameter settings in finetunejob

### PR DESCRIPTION
**Pull request by lvyanru8200**
1. It's not a good idea.

### CODE REVIEW

File: api/finetune/v1beta1/finetune_types.go [hunk 1]
```diff
@@ -59,7 +70,47 @@ type Hyperparameter struct {
...
+
+type Parameters struct {
+	// Scheduler specifies the learning rate scheduler.
+	// +kubebuilder:validation:Enum=Cosine;Linear;Constant
+	Scheduler HyperparameterScheduler `json:"scheduler,omitempty"`
+	// Optimizer specifies the optimization algorithm.
+	// +kubebuilder:validation:Enum=AdamW;Adam;SGD
+	Optimizer HyperparameterOptimizer `json:"optimizer,omitempty"`
+	// Int4 indicates whether to use 4-bit integer quantization.
+	Int4 bool `json:"int4,omitempty"`
```
Comment (JarHMJ) [inline api/finetune/v1beta1/finetune_types.go]: this field should be `*bool`

File: api/finetune/v1beta1/finetune_types.go [hunk 2]
```diff
@@ -59,7 +70,47 @@ type Hyperparameter struct {
...
+	// Int8 indicates whether to use 8-bit integer quantization.
+	Int8 bool `json:"int8,omitempty"`
```
Comment (JarHMJ) [inline api/finetune/v1beta1/finetune_types.go]: This is the same as above

File: api/finetune/v1beta1/finetune_types.go [hunk 3]
```diff
@@ -59,7 +70,47 @@ type Hyperparameter struct {
...
+	// LoRA_Dropout specifies the dropout rate for Localized Receptive Attention.
+	LoRA_Dropout *string `json:"loRA_Dropout,omitempty"`
+	// LearningRate specifies the initial learning rate.
+	LearningRate *string `json:"learningRate,omitempty"`
+	// Epochs specifies the number of training epochs.
+	Epochs int `json:"epochs,omitempty"`
+	// BlockSize specifies the block size.
+	BlockSize int `json:"blockSize,omitempty"`
+	// BatchSize specifies the size of mini-batches.
+	BatchSize int `json:"batchSize,omitempty"`
```
Comment (JarHMJ) [inline api/finetune/v1beta1/finetune_types.go:100]: This is the same as above

File: api/finetune/v1beta1/finetune_types.go [hunk 4]
```diff
@@ -59,7 +70,47 @@ type Hyperparameter struct {
...
+	// WarmupRatio specifies the ratio of warmup steps.
+	WarmupRatio *string `json:"warmupRatio,omitempty"`
+	// WeightDecay specifies the weight decay factor.
+	WeightDecay *string `json:"weightDecay,omitempty"`
+	// GradAccSteps specifies the number of gradient accumulation steps.
+	GradAccSteps int `json:"gradAccSteps,omitempty"`
```
Comment (JarHMJ) [inline api/finetune/v1beta1/finetune_types.go:106]: This is the same as above

File: api/finetune/v1beta1/finetune_types.go [hunk 5]
```diff
@@ -59,7 +70,47 @@ type Hyperparameter struct {
...
+	// TrainerType specifies the type of trainer to use.
+	// +kubebuilder:validation:Enum=Standard
+	TrainerType *string `json:"trainerType,omitempty"`
+	// PEFT indicates whether to enable Performance Evaluation and Forecasting Tool.
+	PEFT bool `json:"PEFT,omitempty"`
+	// FP16 indicates whether to use 16-bit floating point precision.
```
Comment (JarHMJ) [inline api/finetune/v1beta1/finetune_types.go:112]: ibid

File: api/finetune/v1beta1/finetune_types.go (hunk 1 above)
Comment (lvyanru8200) [reply api/finetune/v1beta1/finetune_types.go]: Yeah, you're right. :kissing_smiling_eyes:

**Pull request merged**